src/test/java/org/broadinstitute/hellbender/tools/copynumber/PostprocessGermlineCNVCallsIntegrationTest.java

```diff
@@ -88,6 +94,11 @@ private void runToolForSingleSample(final List<String> callShards,
                     segmentsOutputVCF.getAbsolutePath());
         }
 
+        if (denoisedCopyRatiosOutput != null) {
```

Comment:
Now that `denoisedCopyRatiosOutput` is required, do we need this statement?  For that matter, should we update other code/documentation here to reflect that all outputs are non-optional?

Reply:
Yep, removed both checks